common/src/test/scala/uk/ac/wellcome/utils/TryBackoffTest.scala

```diff
@@ -64,17 +64,17 @@ class TryBackoffTest
     f.calls.length shouldBe finalLength
   }
 
-  it(
```

Comment:
Probably better to mark this test as [quarantined](http://www.scalatest.org/user_guide/tagging_your_tests) rather than comment it out.